cmd: missing state (#1740)
iterative/dvc.org: content/docs/command-reference/status.md

  storage.
- _deleted_ means that the file/directory doesn't exist in the cache, but exists
  in remote storage.
- _missing_ means that the file/directory doesn't exist neither in cache,

>   neither in remote storage.

Reviewer: ```suggestion
  nor in remote storage.
```

> For either _new_ and _deleted_ data, the cache (subset determined by the current

Reviewer: ```suggestion
For _new_ and _deleted_ data, the cache (subset determined by the current
```

Reviewer: Hopefully this doesn't break formatting...

> For _new_ and _deleted_ data, the cache (subset determined by the current
> workspace) is different from remote storage. Bringing the two into sync requires

Reviewer: ```suggestion
For _new_ and _deleted_ data, the cache is different from remote storage.
Bringing the two into sync requires
```

Reviewer: That part was too confusing.

Reviewer: This one will definitely break the formatting...

`dvc pull` or `dvc push`. For the typical process to update the workspace, see
[Sharing Data And Model Files](/doc/use-cases/sharing-data-and-model-files).

> For _missing_ data both cache and remote storage are out of date. Probably
> someone on another machine didn't execute `dvc push` after adding new data. You
> should run `dvc add` or `dvc repro` to rid off _missing_ state.

Reviewer: The intro here is repetitive with the description in the bullet above. Also, I don't think `dvc add` wouldn help you here. Suggestion:

```suggestion
For _missing_ data, there's nothing to retrieve from storage. This can happen
for example in fresh <abbr>DVC repository</abbr> clones if the data wasn't
uploaded from the original repo, or after certain uses of `dvc gc`. You can try
`dvc repro` to regenerate the output locally, and `dvc push` remotely after that.
```
> Paragraph may needs formatting

Reviewer: OK this is the only one left. I'll just commit it and merge the restyled PR to save you time, Nik.

- `--show-json` - prints the command's output in easily parsable JSON format,
  instead of a human-readable table.

- `--all-commits` - same as `-a` or `-T` above, but applies to _all_ Git commits
  as well as the workspace. Useful for comparing cache content for the entire

>   existing commit history of the project.

Reviewer: Thanks for the format fix!

```dvc
$ dvc status
data.csv.dvc:
        changed deps:

>                 update available:   data.csv (different/repo/location)

Reviewer: Please un-do unrelated spacing character changes.

>         changed deps:
>                 update available:   data.csv (different/repo/location)

Reviewer: I'll fix this last one. Please just check that your IDE doesn't do it again 🙂 
```suggestion
	changed deps:
		update available:   data.csv (different/repo/location)
```
